components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -14,6 +15,42 @@ import "components/automate-grpc/protoc-gen-policy/api/annotations.proto";
 import "components/automate-grpc/protoc-gen-policy/iam/annotations.proto";
 
 service Policies {
+  /*
+  Create a new policy
+
+  Creates a new IAM policy used to control permissions in Automate.
+  A policy is composed of one or more statements that grant permissions to a set of members.
+  Each statement contains a role as well as a list of projects.
+
+  The role defines a set of actions that the statement is scoped to.
+  The project list defines the set of resources that the statement is scoped to.
+  Pass `"projects": ["*"]` to scope a statement to every project.
+
+  The policy's top-level projects list defines which project(s) the policy belongs to, for filtering policies by their projects,
+  whereas the statement level projects list defines which project(s) the statement applies to.
+
+  This example creates a new policy in the (unassigned) project that grants the viewer role
```

Comment:
Should clearly delineate such objects, here and throughout:
```suggestion
  This example creates a new policy in the `(unassigned)` project that grants the `viewer` role
```

Reply:
folks, do users need to know that unassigned with implemented as a project internally? if not, id prefer to not be referring to unassigned as a project.

Reply:
Good point, @susanev . So I will amend my comment above:
```suggestion
  This example creates a new policy not associated with any project (because the top-level `projects` property is empty) that grants the `viewer` role
```